docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -64,110 +232,187 @@ POST /transactions/
    .. literalinclude:: samples/post-tx-response.http
       :language: http
 
-   :statuscode 201: A new transaction was created.
-   :statuscode 400: The transaction was invalid and not created.
+   :resheader Content-Type: ``application/json``
+   :resheader Location: As the transaction will be persisted asynchronously, an endpoint to monitor its status is provided in this header.
+
+   :statuscode 202: The pushed transaction was accepted in the ``BACKLOG``, but the processing has not been completed.
+   :statuscode 400: The transaction was malformed and not accepted in the ``BACKLOG``.
 
 
-GET /transactions/{tx_id}/status
+Statuses
 --------------------------------
 
-.. http:get:: /transactions/{tx_id}/status
+.. http:get:: /statuses?tx_id={tx_id}|block_id={block_id}
```

Comment:
I think the pipe (`|`) is confusing here; maybe it'd be nicer just to have two examples?

E.g.:

```
http:get:: /statuses?tx_id={tx_id}
http:get:: /statuses?block_id={block_id}
```